.github/workflows/docker.yml

```diff
@@ -0,0 +1,67 @@
+# The present workflow was made based on the following references:
+# - https://evilmartians.com/chronicles/build-images-on-github-actions-with-docker-layer-caching
+# - https://github.com/docker/build-push-action/blob/master/docs/advanced/cache.md
+---
+name: Docker
+
+on: push
+
+concurrency:
+  group: docker-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-buildx-action@v1
+      - uses: actions/cache@v3
+        with:
+          key: v1-${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: v1-${{ runner.os }}-buildx-
+          path: /tmp/.buildx-cache
```

Comment:
@sestrella I don't think we'll push a tags to create new releases very frequently. Out of curiosity, how long is the cache available to be used in GH Actions?

Reply:
@CristhianMotoche here are some insights:

**Without cache:**

<img width="331" alt="image" src="https://user-images.githubusercontent.com/2049686/162779976-d1fa6414-b709-4c09-855e-e955d184d34f.png">

**With cache:**

<img width="406" alt="image" src="https://user-images.githubusercontent.com/2049686/162780047-19ffe1eb-9abe-46e8-b090-d4ffef750c5d.png">

Reply:
Ignore this comment, too.